### ISSUE TITLE
Tracking laptops with changing ip addresses

### ISSUE THREAD

**Issue by jasonmarkhoffman** [veyon/veyon#399]
Just wondering if anybody has found a way to track student laptops where they have changing ip addresses. Mac addresses are included but do not work on their own,

Windows 10, UVPN disabled

**Comment by ldlx** [comment]
You are connecting by hostname? How often do you clear the DNS cache at the computer running Veyon master and at the DNS server? How often do the Veyon clients re-register at the DNS server?

**Comment by Emalton** [comment]
Yes use hostnames - also check to ensure that your DNS server is working properly too.
